### PR TITLE
Add values.yaml for DREAM & ATBuilding on summit

### PR DESCRIPTION
**Pull request by bbrondel**

### CODE REVIEW

File: apps/auxtel/values-summit.yaml [hunk 1]
```diff
@@ -10,3 +10,6 @@ cscs:
   - atspectrograph
   - atheaderservice
   - pmd1
+spec:
+  source:
+    targetRevision: tickets/DM-47527
```
Comment (tribeiro) [inline apps/auxtel/values-summit.yaml:15]: do you really want to commit with the target being a ticket branch? I think probably not. Also, I think this is something you can edit easily on argo-cd

File: apps/eas/values-summit.yaml [hunk 2]
```diff
@@ -41,3 +41,6 @@ renameMap:
   ess204: auxtel-ess204
   ess301: calibhill-ess301
   ess302: earthquake-ess302
+spec:
+  source:
+    targetRevision: tickets/DM-47527
```
Comment (tribeiro) [inline apps/eas/values-summit.yaml:46]: same comment as above.

**Review by tribeiro — APPROVED** [review]

**Review by tribeiro — COMMENTED** [review]
Make sure you add `atbuilding` to the list of auxtel apps at the summit [here](https://github.com/lsst-ts/argocd-csc/blob/main/apps/auxtel/values-summit.yaml), and `dream` to the list of apps in the obssys apps at the summit [here](https://github.com/lsst-ts/argocd-csc/blob/main/apps/obssys/values-summit.yaml).

**Review by tribeiro — APPROVED** [review]

**Comment by mareuter** [comment]
PR is missing the addition of those CSCs to both the kafka-producers and the love-producer applications for the summit configuration.

**Pull request merged**